DatadogSessionReplay/Sources/Processor/SRDataModelsBuilder/WireframesBuilder.swift

```diff
@@ -98,6 +98,7 @@ public class SessionReplayWireframesBuilder {
         clip: SRContentClip? = nil,
         textColor: CGColor? = nil,
         font: UIFont? = nil,
+        fontSize: CGFloat? = nil,
```

Comment:
I don't entirely like the fact that it's now implicitly either `fontSize` or `font.size`. Maybe we could rename to `fontSizeOverride` ?